frontend/src/components/pages/RegistrantExperience/Waiver/waiverTypes.tsx

```diff
@@ -0,0 +1,43 @@
+import { Waiver, WaiverClause } from "../../../../types/AdminTypes";
```

Comment:
Can this also go in the `types/` directory?

Reply:
yep!